### PR TITLE
[docs] Add a note about what we do if a browser hangs

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @AndreyBelym @AlexKamaev @kirovboris 

https://github.com/DevExpress/testcafe/issues/1815

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
phhhhhhh... even if I create a new pull request, the same tests as in the previous pull request run. Maybe we need to tweak the @testcafe-build-bot? Or this is not because of it?

**Comment by AlexKamaev** [comment]
may be If browser does not respond?

**Comment by AlexKamaev** [comment]
`not respond` does not mean `hang`, but `hang` means `not respond`

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -58,6 +58,9 @@ testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
 > Inactive tabs and minimized browser windows switch to a lower resource consumption mode
 > where tests do not always execute correctly.
 
+If a browser stops responding while it executes tests, TestCafe restarts the browser and tries to continue tests in a new window.
+The test during which the browser stopped responding is reported as failed.
```
Comment (AlexKamaev) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: The test will not be marked as failed in described case. TestCafe will try to restart it

**Pull request merged**